docker-compose.storage.yml

```diff
@@ -45,6 +45,13 @@ services:
       - ME_CONFIG_MONGODB_ADMINPASSWORD=admin
       - ME_CONFIG_MONGODB_URL="mongodb://admin:admin@mongo:27017"
 
+  postgres:
+    image: postgres:16
+    environment:
+      POSTGRES_PASSWORD: password
```

Comment:
Non-blocking: I think we can just skip the password and use `POSTGRES_HOST_AUTH_METHOD: trust`.

Reply:
Additionally, there should be no need for passwords on the AWS instance, it's supposed to be role-based

Reply:
I didn't bother with the more "advanced" configuration since you need a connect string anyway, but updated in 041a73a